src/main/java/com/coveo/spillway/storage/AsyncLimitUsageStorage.java

```diff
@@ -78,6 +78,14 @@ public AsyncLimitUsageStorage(LimitUsageStorage wrappedLimitUsageStorage) {
     return wrappedLimitUsageStorage.debugCurrentLimitCounters();
   }
 
+  public void shutdownStorage() {
+    executorService.shutdown();
```

Comment:
well the shutdown method is only to stop gracefully the sync between the local and the remote storage, it won't actually stop the remote storage